R/ui.R

```diff
@@ -35,13 +33,18 @@ assess_object <- function(og, butchered) {
   if (is.null(mem)) {
     cli::cli_alert_danger("No memory released. Do not butcher.")
   } else {
-    cli::cli_alert_success("Memory released: {.val {mem}}")
-    if (!is.null(disabled)) {
-      cli::cli_alert_danger("Disabled: {.code {disabled}}")
-    }
-    if (length(class_added) == 0) {
-      class_name <- "butchered"
-      cli::cli_alert_danger("Could not add {.cls {class_name}} class")
+    abs_mem <- format(abs(mem), big.mark = ",", scientific = FALSE)
+    if (mem < 0) {
+      cli::cli_alert_danger("The butchered object is {.val {abs_mem}} larger than the original. Do not butcher.")
```

Comment:
With this PR, I see:

```
> res_via_func <- coxph_formula_butchered(example_data)
✖ The butchered object is "920 B" larger than the original. Do not butcher.
```

(with the blue formatting for 920 B). The quotes caught my eye--perhaps we could use `.field` or a straight up glue substitution here?

Reply:
Oh, I see that `Memory released: "533.24 kB"` uses this pattern too. Maybe this is fine!

Reply:
Something about the phrasing this message feels strange to me but maybe that's the purpose—raise an eyebrow in this weird (maybe coxph-specific?) situation.

Reply:
I agree with you that the BLUE plus QUOTES looks weird. What is I change them both to just `.field`? That makes them green with no quote marks.

Reply:
I'm game for that!